e2e-tests/Makefile

```diff
@@ -1,17 +1,45 @@
+# Load environment variables from .env file
+-include .ci/.env
+export
+
 SHELL := /bin/bash
+TEST ?= cypress
+SERVER ?= self-hosted
+
+# Define a conditional default value for ENABLED_DOCKER_SERVICES
+ifeq ($(TEST),cypress)
+    ENABLED_DOCKER_SERVICES ?= postgres inbucket minio openldap elasticsearch keycloak
```

Comment:
I initially thought to make this all running so that in local dev (dev-first approach), Dev has all the required services running by Cypress which can accommodate any of the entire tests. That would be especially helpful for external (first-time) contributors. But I'm fine to make this "smoketests-first" and clearly document the default and usage.

Reply:
Done, `postgres inbucket` is now the default `ENABLED_DOCKER_SERVICES` and also dded notes in readme.